content/en/docs/setup/quickstart.md

```diff
@@ -12,8 +12,8 @@ If you want to install Spinnaker on Lightweight Kubernetes (K3s) for proofs of c
 
 Options for installing Spinnaker in Kubernetes:
 
+- [Spinnaker Helm Chart](https://github.com/OpsMx/spinnaker-helm) is an open source helm chart to install/configure spinnaker in GitOps/Non-GitOps style
```

Comment:
I'm not sure what GitOps/Non-GitOps style means here.
```suggestion
- [Spinnaker Helm Chart](https://github.com/OpsMx/spinnaker-helm) is an open source helm chart to install/configure spinnaker.
```

Reply:
Removed Gitops/Non-Gitops style phrase